docs/source/FAQ/data_manager_migration.rst

```diff
@@ -0,0 +1,138 @@
+
+Previously, sharing a Dataset from one Project to another was a 2-step process: first publishing the Dataset to a Team, then pulling the Dataset into the new project.
+Now that all Datasets are assets of teams, sharing within a team is unnecessary.
+All of the `publish`, `un_publish`, and `pull_in_resource` endpoints, when applied to Datasets will undergo deprecation.
+To be precise, the following calls will return a deprecation warning version for Citrine Python versions 3.4 and above, and be removed in version 4.0:
+
+.. code-block:: python
+
+    # Publishing a Dataset to a Team will do nothing once Data Manager is activated
+    project.publish(resource=dataset)
+
+    # Un-publishing a Dataset will similarly be a no-op with Data Manager activated
+    project.un_publish(resource=dataset)
+
+    # Pulling a Team into a Project can still be done with Data Manager activated, but not
+    # recommended and will be deprecated
+    project.pull_in_resource(resource=dataset)
+
+**Between Teams**
+
+Sharing a Dataset from one project to another where those projects are in different Teams was a 3-step process:
+publishing to the Team, sharing from one Team to another, then pulling into a Project.
+With Data Manager, only the sharing action is needed.
+
+Previous code for sharing My Dataset from Project A in Team A to eventually use in a Training Set
+in Project B in Team B:
+
+.. code-block:: python
+
+    project_a.publish(resource=my_dataset)
+    team_a.share(
+        resource=my_datset,
+        target_team_id=team_b.uid,
+    )
+    project_b.pull_in_resource(resource=my_dataset)
+
+Is now:
+
+.. code-block:: python
+
+    team_a.share(
+        resource=my_datset,
+        target_team_id=team_b.uid,
+    )
+
+Project-based Collections
+-------------------------
+
+As Datasets are now assets of Teams, typical ways to `list()`, `get()`, or otherwise manipulate Datasets or data objects within a Project will undergo a deprecation cycle.
+As of v3.4, these endpoints will still work as usual with a deprecation warning, but will be removed in v4.0.
+It is therefore recommended to migrate your code from all project-based listing endpoints as soon as possible
+to adhere to supported patterns and avoid any costly errors.
+
+The following endpoints will return a return a deprecation warning version for Citrine Python versions 3.4 and above, and be removed in version 4.0.
+Moreover, they will not reference Datasets or their contents that are registered after Data Manager has been activated:
+
+.. code-block:: python
+
+    # Listing Datasets or their Contents (such as MaterialSpecs or ProcessTemplates) from a Project
+    project.[datasets/gemd_objects].list()
+
+    # Getting Datasets or GEMD Assets via their UID and a Project
+    project.[datasets/gemd_objects].get(UID)
+
+    # Doing any operations (updating, deleting, dumping, etc.) to Datasets or GEMD Assets via a
+    # Project collection
+    project.[datasets/gemd_objects]....
```

Comment:
You should never have been able to do this along a project-based route for GEMD objects.  It always required a dataset (maybe delete was an exception? Definitely true for update).  In the past, that returned a simple error. @pacdaemon and @anoto-moniz might have accidentally made something broken when they added their deprecations there.



```suggestion

    # Doing any operations (updating, deleting, etc.) to Datasets via a Project collection
    project.datasets.delete(uid)
```

Reply:
I thought the actions were just read-only if only the project was reference. E.g. .list() or .get() would work, but not register or update

Reply:
Addressed, though I used "update" as an example since we use "delete before. Still not sure about whether or not we allow update or other things from the team-level endpoints.

Reply:
@kroenlein To clarify, is there something in the SDK code that makes you think something's broken? Or are you just saying this line of the documentation was wrong?

Reply:
I ran it locally and got a giant splat of exception handling that did not indicate the real problem.  However, I then ran the same code with v3.2.11 and got the same error, so if we have more elegant handling in the past, that was last in a previous release.